commit f791c862d81b027c9da47005e6b8fb7bb1f9687f
Author: Daniel Orstadius <daniel.orstadius@nokia.com>
Date:   Tue Oct 19 17:41:27 2010 +0300

    audio/device.h: use lower case in profile UUIDs
    
    BlueZ seems to use lower case when emitting signals containing
    UUIDs retrived from a remote device.
    
    The definitions affected by this patch look to be used for example
    if the remote connects an audio profile which BlueZ has not
    discovered (perhaps it had not yet browsed the service records of
    the remote).
    
    In that case the format of the stored and emitted UUID should be
    consistent with the UUIDs received during service discovery.

diff --git a/audio/device.h b/audio/device.h
index 6ec3938..35af788 100644
--- a/audio/device.h
+++ b/audio/device.h
@@ -22,21 +22,21 @@
  *
  */
 
-#define GENERIC_AUDIO_UUID	"00001203-0000-1000-8000-00805F9B34FB"
+#define GENERIC_AUDIO_UUID	"00001203-0000-1000-8000-00805f9b34fb"
 
-#define HSP_HS_UUID		"00001108-0000-1000-8000-00805F9B34FB"
-#define HSP_AG_UUID		"00001112-0000-1000-8000-00805F9B34FB"
+#define HSP_HS_UUID		"00001108-0000-1000-8000-00805f9b34fb"
+#define HSP_AG_UUID		"00001112-0000-1000-8000-00805f9b34fb"
 
-#define HFP_HS_UUID		"0000111E-0000-1000-8000-00805F9B34FB"
-#define HFP_AG_UUID		"0000111F-0000-1000-8000-00805F9B34FB"
+#define HFP_HS_UUID		"0000111e-0000-1000-8000-00805f9b34fb"
+#define HFP_AG_UUID		"0000111f-0000-1000-8000-00805f9b34fb"
 
-#define ADVANCED_AUDIO_UUID	"0000110D-0000-1000-8000-00805F9B34FB"
+#define ADVANCED_AUDIO_UUID	"0000110d-0000-1000-8000-00805f9b34fb"
 
-#define A2DP_SOURCE_UUID	"0000110A-0000-1000-8000-00805F9B34FB"
-#define A2DP_SINK_UUID		"0000110B-0000-1000-8000-00805F9B34FB"
+#define A2DP_SOURCE_UUID	"0000110a-0000-1000-8000-00805f9b34fb"
+#define A2DP_SINK_UUID		"0000110b-0000-1000-8000-00805f9b34fb"
 
-#define AVRCP_REMOTE_UUID	"0000110E-0000-1000-8000-00805F9B34FB"
-#define AVRCP_TARGET_UUID	"0000110C-0000-1000-8000-00805F9B34FB"
+#define AVRCP_REMOTE_UUID	"0000110e-0000-1000-8000-00805f9b34fb"
+#define AVRCP_TARGET_UUID	"0000110c-0000-1000-8000-00805f9b34fb"
 
 /* Move these to respective .h files once they exist */
 #define AUDIO_SOURCE_INTERFACE		"org.bluez.AudioSource"
